packages/fluentui/CHANGELOG.md

```diff
@@ -7,7 +7,6 @@
   ### Features
   ### Performance
   ### Documentation
```

Comment:
This looks like a bad merge, I don't expect any changes in this file for this PR...

Reply:
this is from a bad merge i think .. looking into resolving and reverting

Reply:
redid the merge and should be up to date